db/seeds.rb

```diff
@@ -5,3 +7,8 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Card.create(original_text: 'Car', translated_text: 'Автомобиль', review_date: Time.now)
+Card.create(original_text: 'Cat', translated_text: 'Кот', review_date: Time.now)
+Card.create(original_text: 'Dog', translated_text: 'Собака', review_date: Time.now)
+Card.create(original_text: 'City', translated_text: 'Город', review_date: Time.now)
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.